python/src/GaussLegendre_doc.i.in

```diff
@@ -9,27 +9,87 @@ Available constructors:
 Parameters
 ----------
 dimension : int, :math:`dimension>0`
-    The dimension of the functions to integrate. The default discretization is *GaussLegendre-DefaultMarginalIntegrationPointsNumber* in each dimension, see :class:`~openturns.ResourceMap`.
+    The input dimension of the functions to integrate.
 discretization : sequence of int
     The number of nodes in each dimension. The sequence must be non-empty and must contain only positive values.
+    
+    The default discretization is *GaussLegendre-DefaultMarginalIntegrationPointsNumber* in each dimension, see :class:`~openturns.ResourceMap`.
 
 Notes
 -----
-The Gauss-Legendre algorithm enables to approximate the definite integral:
+We consider a function :math:`f: \Rset^d \mapsto \Rset^p` and a domain :math:`\set{D} = [\vect{a}, \vect{b}]`. The Gauss-Legendre algorithm enables to approximate the definite integral:
 
 .. math::
+    :label: intInit
 
-    \int_{\vect{a}}^\vect{b} f(\vect{t})\di{\vect{t}}
+    \int_{\vect{a}}^{\vect{b}} f(\vect{t})\di{\vect{t}}
 
 
-with :math:`f: \Rset^d \mapsto \Rset^p`, :math:`\vect{a}, \vect{b}\in\Rset^d` using a fixed tensorized Gauss-Legendre approximation:
+using a fixed tensorized Gauss-Legendre approximation. The approximation is:
 
 .. math::
+    :label: approxInt
 
-    \int_{\vect{a}}^\vect{b} f(\vect{t})\di{\vect{t}} = \sum_{\vect{n}\in \cN}\left(\prod_{i=1}^d w^{N_i}_{n_i}\right)f(\xi^{N_1}_{n_1},\dots,\xi^{N_d}_{n_d})
+    \int_{\vect{a}}^{\vect{b}} f(\vect{t}) \di{\vect{t}} = \sum_{n_1=1}^{N_1} \dots  \sum_{n_d=1}^{N_d}\left( \prod_{i=1}^d w^{N_i}_{n_i}\right) f(\xi^{N_1}_{n_1},\dots,\xi^{N_d}_{n_d})
 
-where :math:`\xi^{N_i}_{n_i}` is the :math:`n_i` th node of the :math:`N_i` -points Gauss-Legendre 1D integration rule and :math:`w^{N_i}_{n_i}` the associated weight.
+where :math:`\xi^{N_i}_{n_i}` is the :math:`n_i` -th node computed from the :math:`N_i` -points Gauss-Legendre 1D integration rule and :math:`w^{N_i}_{n_i}` the associated weight.
 
+To use that approximation, the integration variables are scaled to the domain :math:`[0,1]^d` with the mapping function~:math:`\varphi`:
```

Comment:
There is a problem with function~:math:`\varphi`

Reply:
yes